src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+be flushed with the element snap area start edge. Similarly, the `end` and
+`center` alignments mean that the scroll container snapport end edge or center
+should be flushed with the element snap area end edge or center.
+
+[Snapport](https://drafts.csswg.org/css-scroll-snap/#scroll-padding) is the area
+of the scroll container to which the snap areas are aligned. By default it is
+the same as the visual viewport of the scroll container but it can be adjusted
+using `scroll-padding` property.
+
+
+
+<figure>
+    <video src="/web/updates/images/2018/07/css-scroll-snap/interactive-alignment.mp4"
+    autoplay loop muted playsinline alt="Alignment demo"> </video>
+  <figcaption>
+    Example of a various alignments on horizontal scrolling axis.
+ </figcaption>
+</figure>
+
+
+
+The following examples illustrate how these concepts can be used in practice.
+
+
+### Example - Horizontal gallery
+
+A common use case for scroll snapping is an  image carousel. For example, to
+create a horizontal image carousel that snaps to each image as you scroll, we
+can specify the scroll container to have a mandatory ``scroll-snap-type`` on the
+horizontal axis.  set each image to ``scroll-snap-align: center`` to ensure that
+the snapping centers the image within the carousel.
+
+
+```
+<style>
+#gallery {
+  scroll-snap-type: x mandatory;
+  overflow-x: scroll;
+  display: flex;
+}
+
+#gallery img {
+   scroll-snap-align: center;
+}
+</style>
+
+<div id="gallery">
+  <img src="cat.jpg">
+  <img src="dog.jpg">
+  <img src="another_cute_animal.jpg">
+</div>
+```
+
+Because snap positions are associated with an element, the snapping algorithm
+can be smart about when and how it snaps given the element and the scroll
+container size. For example, consider the case where one image is larger than
+the carousel. A naïve snapping algorithm may prevent the user from panning
+around to see the full image. But the
+[specification](https://drafts.csswg.org/css-scroll-snap/#snap-overflow)
+requires implementations to detect this case and allow the user to freely scroll
+around within that image only snapping at its edges.
+
+
+<figure>
+  <a href="/web/updates/images/2018/07/css-scroll-snap/gallery-page.mp4"
+     target="_blank">
+    <video src="/web/updates/images/2018/07/css-scroll-snap/gallery-page.mp4"
+           autoplay loop muted playsinline controls alt="Gallery page demo" height="600"></video>
+  </a>
+  <figcaption>
+    <a href="https://snap.glitch.me/carousel.html"
+       target="_blank">View demo</a> |
+    <a href="https://glitch.com/edit/#!/snap?path=carousel.html:1:0"
+       target="_blank">Source</a>
+  </figcaption>
+</figure>
+
+
+### Example - Journeyed product page
+
+Another common case that can benefit from scroll snapping are pages with
+multiple logical sections that are vertically scrolled through, e.g., a typical
+product page. ``scroll-snap-type: y proximity;` is a `more natural fit for cases
```

Comment:
Double back ticks at the start, but not at the end, use only single backticks.

Reply:
Done